src/include/distributed/connection_management.h

```diff
@@ -28,8 +28,8 @@
 /* used for libpq commands that get an error buffer. Postgres docs recommend 256. */
 #define ERROR_BUFFER_SIZE 256
 
-/* application name used for internal connections in Citus */
-#define CITUS_APPLICATION_NAME "citus"
+/* application name prefix used for internal connections in Citus */
+#define CITUS_APPLICATION_NAME_PREFIX "citus gpid="
```

Comment:
can there be cases where we connect as "citus", but without a gpid (e.g. deadlock detector?)

should we just use "citus" in that case? (or a custom name?)